### PR TITLE
fix(curriculum): cat photo app workshop step 5 to 7

### LINKED ISSUE #56309: Update steps 5-7 of cat photo workshop to use specific assert methods where appropriate
### Describe the Issue

We recently updated the guidelines for writing assertions here 
https://contribute.freecodecamp.org/how-to-work-on-coding-challenges/#writing-tests

For consistency and matching what is in the docs, we should slowly start to update the tests for the workshops and labs in the new frontend cert.

For this issue, just focus on updating the tests for steps 5-7 of the cat photo app workshop.
We don't want to have a large PR that updates a whole bunch of tests in a bunch of steps and makes it harder to review. 

Here is the link to the workshop:
https://github.com/freeCodeCamp/freeCodeCamp/tree/main/curriculum/challenges/english/25-front-end-development/workshop-cat-photo-app

The tests should be updated, where appropriate, to use specific methods like `assert.equal`, `assert.exists`, `assert.match`, etc instead of just `assert`.

Here is the complete API reference 
https://www.chaijs.com/api/assert/#method_exists

Just focus on the new frontend cert and don't worry about updating the RWD cat photo app since that will become legacy anyway.

### Affected Page

multiple

### Your code

N/A

### Expected behavior

see explanation above

### Screenshots

_No response_

### System

- Device: [e.g. iPhone 6, Laptop]
- OS: [e.g. iOS 14, Windows 10, Ubuntu 20.04]
- Browser: [e.g. Chrome, Safari]
- Version: [e.g. 22]

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by gagan-bhullar-tech**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #56309

### CODE REVIEW

File: curriculum/challenges/english/25-front-end-development/workshop-cat-photo-app/6690bd5ad96139157eedf697.md [hunk 1]
```diff
@@ -25,70 +25,68 @@ The `h1` element, `h2` element and the comment are indented two spaces more than
...
+assert.exists(document.querySelector('h2'));
+assert.match(code, /<\/h2\>/);
+assert.equal(document.querySelector('h2').innerText.toLowerCase(),'cat photos')
 ```
 
 You should not add the `ul` or `li` elements from the example.
 
 ```js
-assert(
+assert.isTrue(
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/workshop-cat-photo-app/6690bd5ad96139157eedf697.md:36]: can you make this 2 separete asserts using `assert.notExists`? it makes more sense with what is tested

File: curriculum/challenges/english/25-front-end-development/workshop-cat-photo-app/6690be42942f421648b119eb.md [hunk 2]
```diff
@@ -16,19 +16,19 @@ Add an `img` element below the `p` element. At this point, no image will show up
...
 ```js
-assert(!code.match(/<\/img\>/));
+assert.notMatch(code, /<\/img\>/);
 ```
 
 You should only have one `img` element. Remove any extras.
 
 ```js
-assert(document.querySelectorAll('img').length === 1);
+assert.equal(document.querySelectorAll('img').length, 1);
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/workshop-cat-photo-app/6690be42942f421648b119eb.md:31]: here you can use `assert.lengthOf`

**Review by majestic-owl448 — COMMENTED** [review]
the title of the workshop is cat photo app, please change to that so that it is understandable for what workshop this issue is for

**Comment by gagan-bhullar-tech** [comment]
@ilenia-magoni updated PR

File: curriculum/challenges/english/25-front-end-development/workshop-cat-photo-app/6690be42942f421648b119eb.md [hunk 3]
```diff
@@ -16,19 +16,19 @@ Add an `img` element below the `p` element. At this point, no image will show up
...
 ```js
-assert(!code.match(/<\/img\>/));
+assert.notMatch(code, /<\/img\>/);
 ```
 
 You should only have one `img` element. Remove any extras.
 
 ```js
-assert(document.querySelectorAll('img').length === 1);
+assert.lengthOf(document.querySelectorAll('img').length, 1);
```
Comment (gikf) [inline curriculum/challenges/english/25-front-end-development/workshop-cat-photo-app/6690be42942f421648b119eb.md:31]:
```suggestion
assert.lengthOf(document.querySelectorAll('img'), 1);
```

**Review by gikf — APPROVED** [review]

**Review by jdwilkin4 — APPROVED** [review]

**Pull request merged**